docs/usage/dependency-dashboard.md

```diff
@@ -0,0 +1,152 @@
+
+To disable the Dependency Dashboard, set `dependencyDashboard` to `false`.
+
+```json
+{
+  "extends": ["config:base"],
+  "dependencyDashboard": false
+}
+```
+
+## Usecases
+
+This section explains some common usecases where having the Dependency Dashboard can help.
+
+### Visibility into rejected/deferred updates
+
+Renovate's Dependency Dashboard shows a overview of all updates that are still "to do".
+
+If you close a update PR from Renovate, the Dashboard will list this update.
+If you later change your mind about the update, you can get a new PR by clicking the corresponding checkbox on the dashboard.
+
+### Show errored out updates
+
+<!-- TODO: Not sure if showing errored updates/branches require the dashboard? Do we open a new issue to warn about a branch error? -->
+
+### Dependency Dashboard Approval workflow
+
+Sometimes you want Renovate to wait for your approval before creating a update PR.
+You can customize this "wait for approval" behavior however you like best.
+
+At a high level the options are:
+
+- Require approval for _all_ updates
+- Require approval for a type of updates (`major` for example)
+- Require approval for specific packages
+
+You can mix and match these options as well.
+
+#### Require approval for all updates
+
+We do not recommend that you require approval for _all_ updates.
+When you require prior approval, you need to check the dashboard issue regularly to check for important updates.
+You'll probably forgot to check often enough, out of sight is out of mind!
+
+Maybe you find Renovate too noisy, and want to opt-out of getting automatic updates whenever they're ready.
+
+In this case, you can tell Renovate to wait for your approval before making any pull requests.
+This means that you have full control over when you get updates.
+
+<!-- TODO: question: do you still get security updates when you tell Renovate to wait for approval for all updates? -->
+
+Make sure you explictly enable the Dependency Dashboard this way have visibility into all pending updates.
+
+To require manual approval for _all updates_, add the `:dependencyDashboard` and the `:dependencyDashboardApproval` presets to the `extends` array in your Renovate configuration file:
+
+```json
+{
+  "extends": [
+    "config:base",
+    ":dependencyDashboard",
+    ":dependencyDashboardApproval"
+  ]
+}
+```
+
+#### Require approval for major updates
+
+Major updates are likely to break tests and/or require manual work before they can be merged.
+So maybe you only want to get major updates when you approve them.
+
+Dependency Dashboard Approval is far superior to disabling major updates because at least you can fully see what's pending on the dashboard, instead of updates being totally invisible.
+
+If you want to require approval for major updates, set `dependencyDashboardApproval` to `true` within a `major` object:
+
+```json
+{
+  "major": {
+    "dependencyDashboardApproval": true
+  }
+}
```

Comment:
We should use a package rule instead the shortcut obejects. They are more clear, as a package rule will override short object settings.

Reply:
I copy/pasted this from https://docs.renovatebot.com/configuration-options/#dependencydashboardapproval

> If you want to require approval for major updates, set `dependencyDashboardApproval` to `true` within a `major` object:

If this example is bad, we should fix it over there as well. 😄

Reply:
It's not bad, but generally using package rules is best practise. @rarkins WDYT?